docs/common/reproducible-build.md

```diff
@@ -1,64 +1,111 @@
 # Reproducible build
 
-We want to invite the wider community to participate in the verification of
-the firmware built by SatoshiLabs. With reasonable effort you should be able to
-build the firmware and verify that it's identical to the official firmware.
+We want to invite the wider community to participate in the verification of the firmware
+built by Trezor Company. With reasonable effort you should be able to build the firmware
+and verify that it's identical to the official firmware.
 
-Trezor Firmware uses [Nix](https://nixos.org/), [Poetry](https://python-poetry.org/)
-and [Cargo](https://doc.rust-lang.org/cargo/) to make the build environment
-deterministic. We also provide a Docker-based script so that the build can be
-performed with a single command on usual x86 Linux system.
+Trezor Firmware uses [Nix](https://nixos.org/), [Poetry](https://python-poetry.org/) and
+[Cargo](https://doc.rust-lang.org/cargo/) to make the build environment deterministic.
+We also provide a Docker-based script so that the build can be performed with a single
+command on usual x86 Linux system.
 
 ## Building
 
 First you need to determine which *version tag* you want to build:
 * for Trezor One it is `legacy/vX.Y.Z`, e.g. `legacy/v1.10.3`,
-* for Trezor Model T it is `core/vX.Y.Z`, e.g. `core/v2.4.2`.
+* for newer models, it is `core/vX.Y.Z`, e.g. `core/v2.4.2`.
 
-Assuming you want to build `core/v2.4.2`:
+Assuming you want to build `core/v2.8.3`:
 
 1. install [Docker](https://www.docker.com/)
 2. clone the firmware repository: `git clone https://github.com/trezor/trezor-firmware.git`
 3. go into the firmware directory: `cd trezor-firmware`
-4. checkout the version tag: `git checkout core/v2.4.2`
-5. run: `bash build-docker.sh core/v2.4.2`
+4. checkout the version tag: `git checkout core/v2.8.3`
+5. run: `bash build-docker.sh core/v2.8.3`
 
 After the build finishes the firmware images are located in:
 * `build/legacy/firmware/firmware.bin` and `build/legacy-bitcoinonly/firmware/firmware.bin` for Trezor One,
-* `build/core/firmware/firmware.bin` and `build/core-bitcoinonly/firmware/firmware.bin` for Trezor Model T.
+* `build/core-<model>/firmware/firmware.bin` and `build/core-<model>-bitcoinonly/firmware/firmware.bin` for later models.
+
+### Model identifiers
+
+You can speed up the build process by adding options to the script:
+
+* `--skip-core` if you are only building for Trezor One,
+* `--skip-legacy --models=A,B,C` to only build for specific model(s) which are not Trezor One.
+
+The following models are supported:
+
+* **`T`** - Trezor Model T
+* **`R`** - Trezor Safe 3 rev.A
```

Comment:
can we change it to T2T1 and T2B1 for TT and TS3 rev.A?

Reply:
not in this PR but we can make a separate issue for it

Reply:
Can you please create an issue? 🙏🏻